src/main/java/com/vishva/Payapp/controller/CreditController.java

```diff
@@ -6,28 +6,30 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import javax.validation.Valid;
 
 @RestController
-@RequestMapping("/")
+@RequestMapping("/home")
 public class CreditController {
 
     @Autowired
     private CreditService creditService;
 
-    @GetMapping("/home")
+    @GetMapping("/")
     public String index() {
         return "Welcome to Payapp!";
     }
 
-    @PostMapping("/home/credit")
-    public ResponseEntity<CreditResponse> getResponse(@RequestBody CreditRequest creditRequest) {
+    @PostMapping("/credit")
+    public ResponseEntity<CreditResponse> creditAmount(@Valid @RequestBody CreditRequest creditRequest) {
 
-        if(creditRequest.getAccId() != null && creditRequest.getAmount() != null) {
-            creditService.amountCredited(creditRequest);
-            return ResponseEntity.ok().build();
-        }
-        else {
-            return ResponseEntity.badRequest().body(null);
+        if(creditRequest.getAccId() == null || creditRequest.getAmount() == null) {
```

Comment:
No idea for this, your @Valid will throw Exception and then you can use @ControllerAdvice for sending the response corresponding to given exception